src/v2/spec.yaml

```diff
@@ -3158,6 +3158,36 @@ paths:
                 $ref: "#/components/schemas/UsersResponse"
         "400":
           $ref: "#/components/responses/400Response"
+  /farcaster/user/power_lite:
+    get:
+      tags:
+        - User
+      summary: Fetch power user FIDs
+      description: Fetches power users and respond in a backwards compatible format to Warpcast's deprecated power badge endpoint.
+      externalDocs:
+        url: https://docs.neynar.com/reference/user-power-lite
+      operationId: user-power-lite
+      parameters:
+        - $ref: "#/components/parameters/ApiKey"
+      responses:
+        "200":
+          description: Successful operation.
+          content:
+            application/json:
+              schema:
```

Comment:
Schema is incorrect, it should look like this

```
schema:
  type: object
  required:
    - result
  properties:
    result:
      type: object
      required:
        - fids
      properties:
        fids:
          type: array
          description: List of Fids
          items:
            $ref: "#/components/schemas/Fid"
```

separating into 2 schema will look like this, this way we know that generated models have desired names 

```
responses:
  "200":
    description: Successful operation.
    content:
      application/json:
        schema:
          $ref: "#/components/schemas/PowerUserLiteResponse"  

Fids:
  type: array
  items:
    $ref: "#/components/schemas/Fid"
  description: List of FIDs

PowerUserLiteResponse:
  type: object
  required:
    - result
  properties:
    result:
      type: object
      required:
        - fids
      properties:
        fids:
          $ref: "#/components/schemas/Fids" 
```

Reply:
> this way we know that generated models have desired names

makes sense, thanks for the context. will update